src/docs/grains/index.md

```diff
@@ -5,8 +5,8 @@ title: Developing a Grain
 
 # Setup
 
-Before you write code to implement a grain class, create a new Class Library project targeting .NET Standard (preferred) or .NET Framework 4.6.1 or higher (if you cannot use .NET Standard due to dependencies).
-Grain interfaces and grain classes can be defined in the same Class Library project or in two different projects for better separation of interfaces from implementation.
+Before you write code to implement a grain class, create a new Class Library project targeting .NET Standard or .Net Core (preferred) or .NET Framework 4.6.1 or higher (if you cannot use .NET Standard or .Net Core due to dependencies).
```

Comment:
.NET should be capitalized

Reply:
Good catch! Now how do I fix it?

Reply:
You can edit the file in your PR. I can edit it, too, so I'll edit it.